sklearn/metrics/_plot/roc_curve.py

```diff
@@ -185,6 +185,17 @@ def plot_roc_curve(estimator, X, y, pos_label=None, sample_weight=None,
         if prediction_method is None:
             raise ValueError('response methods not defined')
 
+    if (pos_label is not None
+            and hasattr(estimator, "classes_")
```

Comment:
All the classifiers have ``classes_``?

Reply:
Third party estimator classes might not be be very strict in implementing the scikit-learn API. I did not want to break those if a simple ducktyping check can avoid introducing an expected change in behavior when upgrading scikit-learn.

Reply:
Actually this is a new function so we do not need to deal with backward compat in this case.